sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/randomExpressions.scala

```diff
@@ -102,6 +102,8 @@ case class Rand(child: Expression) extends RDG with ExpressionWithRandomSeed {
   }
 
   override def freshCopy(): Rand = Rand(child)
+
+  override def sql: String = "rand()"
```

Comment:
Hmmm .. shouldn't we print out the seed when it's explicitly given?

Reply:
Ur, I see... I just want to reomve a random number in a column name when no argument given. I need more time to think of it...

Reply:
I think maybe the current output is useful. Yes the seed was randomly chosen, but you might want to know what it was.

Reply:
Yea, I also think it's important that we can check seeds, but `df.explain` is not enough for checking it? Actually, the other two expression with random seeds (`shuffle` and `uuid`) don't display it in column names;
```
scala> sql("select shuffle(array(1, 2))").show()
+--------------------+
|shuffle(array(1, 2))|
+--------------------+
|              [2, 1]|
+--------------------+

scala> sql("select shuffle(array(1, 2))").explain()
== Physical Plan ==
*(1) Project [shuffle([1,2], Some(894779230406706679)) AS shuffle(array(1, 2))#14]
+- *(1) Scan OneRowRelation[]

scala> sql("select uuid()").show()
+--------------------+
|              uuid()|
+--------------------+
|dde93891-8a95-4e9...|
+--------------------+

scala> sql("select uuid()").explain()
== Physical Plan ==
*(1) Project [uuid(Some(4613707233104825008)) AS uuid()#23]
+- *(1) Scan OneRowRelation[]
```

Reply:
Fair point, consistency is good. I don't feel strongly either way.